adage/adage/config.py.template

```diff
@@ -7,6 +7,10 @@
 # and replace with values tailored to your deployment.
 import random
 
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
True, but see my comment above about incompatibility with `fab` scripts. As long as we keep `CONFIG` settings specific to a deployment scenario this should work fine.